cluster/charts/crossplane/crds/gcp/database/v1alpha1/cloudsqlinstance.yaml

```diff
@@ -55,6 +66,13 @@ spec:
             storageType:
               type: string
             tier:
+              description: MySQL and PostgreSQL use different machine types.  MySQL
```

Comment:
i really like how the comments from types.go are being generated into the CRDs now, that's incredibly useful to have valuable comments from the source code in another possibly more visible place for the users.  Eventually we'll want to generate docs from the source comments as well, but this is a great step!